### ISSUE TITLE
NPM Release 

### ISSUE THREAD

**Issue by akash2004u** [rexrainbow/phaser4-rex-plugins#1]
Hi Rex, 

We are starting a game, and would love to adopt Phaser 4 and RexPlugins Since development. Would love to contribute back in the process.

Can you start pushing Phaser4-Rex-plugings to NPM , so that we can start consuming those ?
Additionally is there an alternative to consume the packages if you are not planning to release those in Alpha / Beta

**Comment by rexrainbow** [comment]
Since phaser4 is not released yet, and changing the design very often. I can not provide a compatible plugins for latest phaser4 currently. Therefore I can't publish npm package now.

**Comment by akash2004u** [comment]
Makes sense, and currently we can work around using install from git URL directly. 
But since Phaser 4 is already in beta and available i was wondering if it makes it easier for plugins to be available too even if in ALPHA state.

**Comment by rexrainbow** [comment]
I will back to phaser4 plugin when the ts definition of my phaser3 plugin are all finished.
